plugins/GigPlayer/GigPlayer.cpp

```diff
@@ -440,50 +430,44 @@ void GigInstrument::play( SampleFrame* _working_buffer )
 				samples = frames / freq_factor + Sample::s_interpolationMargins[m_interpolation];
 			}
 
+			const auto neededCapacity = static_cast<std::vector<SampleFrame>::size_type>(samples);
```

Comment:
`samples` is already `std::size_t` so it looks like there's no need for the static cast. And the `neededCapacity` variable may be unnecessary if you can just use `samples`.

Reply:
same here